src/event-detail/ui/event-detail.controller.js

```diff
@@ -80,17 +78,31 @@ function EventDetail(
 
           var canEditMovies = hasEditMoviesPermission && hasMovieLabel;
 
-          if ($scope.isGodUser) {
-            $scope.permissions = {editing: true, editingMovies: canEditMovies, duplication: true, history: true};
-          } else if (hasPermission) {
-            $scope.permissions = {
-              editing: !event.isExpired(),
+          $scope.isGodUser = _.filter(userPermissions, function(permission) {
+            return permission === RolePermission.GEBRUIKERS_BEHEREN;
+          }).length > 0;
+
+          var offerPermissions = {editing: false, moderate: false, delete: false};
+
+          if (hasPermissions) {
+            if (_.includes(permissions, 'Aanbod bewerken') && (!event.isExpired() || $scope.isGodUser)) {
+              offerPermissions.editing = true;
+            }
+            if (_.includes(permissions, 'Aanbod modereren')) {
+              offerPermissions.moderate = true;
+            }
+            if (_.includes(permissions, 'Aanbod verwijderen'))  {
+              offerPermissions.delete = true;
+            }
```

Comment:
```suggestion
          var canEdit = _.includes(permissions, 'Aanbod bewerken') && (!event.isExpired() || $scope.isGodUser);
          var offerPermissions = {
            editing: canEdit,
            moderate: _.includes(permissions, 'Aanbod modereren'),
            delete: _.includes(permissions, 'Aanbod verwijderen')
          };
```

It could also be written this way